### PR TITLE
WIP: Use property

### LINKED ISSUE #426: warning messages at `using PyPlot`
after the update of PyCall to 1.90.0, starting with 

```
 Warning: `getindex(o::PyObject, s::Symbol)` is deprecated in favor of dot overloading (`getproperty`) so elements should now be accessed as e.g. `o.s` instead of `o[:s]`.
│   caller = __init__() at init.jl:175
```

### PR DESCRIPTION
**Pull request by tkf**
It's mostly `sed`-based edit so maybe I'm missing something.  But it seems it's working...

Closes #426.

### CODE REVIEW

**Comment by stevengj** [comment]
Thanks.  It will have to wait until a new version of PyCall is tagged, however.

**Comment by stevengj** [comment]
Update to REQUIRE PyCall 1.90

File: src/PyPlot.jl [hunk 1]
```diff
@@ -53,34 +55,42 @@ include("init.jl")
...
 
 for (mime,fmt) in aggformats
     @eval function show(io::IO, m::MIME{Symbol($mime)}, f::Figure)
-        if !haskey(pycall(f.o["canvas"]["get_supported_filetypes"], PyDict),
+        if !haskey(pycall(PyObject(f)."canvas"."get_supported_filetypes", PyDict),
                    $fmt)
             throw(MethodError(show, (io, m, f)))
         end
-        f.o["canvas"]["print_figure"](io, format=$fmt, bbox_inches="tight")
+        PyObject(f)."canvas"."print_figure"(io, format=$fmt, bbox_inches="tight")
```
Comment (stevengj) [inline src/PyPlot.jl]: Can be just `f."canvas"....`, I think.

**Comment by stevengj** [comment]
Should be good to merge once AppVeyor is green.

**Comment by tkf** [comment]
Thanks for finishing this up.

**Pull request merged**